src/rmet/metrics.py

```diff
@@ -150,6 +151,38 @@ def hitrate(logits: torch.Tensor, targets: torch.Tensor, k: int = 10, logits_are
     return recall
 
 
+def mean_average_precision(logits: torch.Tensor, targets: torch.Tensor, k: int = 10,
```

Comment:
If I understand correctly, this is still not averaged over users, so the returned tensor will be of shape `(n_samples, )`. So maybe it would be good to call it `average_precision` only?

Reply:
Good point. I was thinking of calling it `mean_average_precision`, because the aggregated, i.e., averaged results would be `MAP` and not `AP`. For correctness, I'll change the name `average_precision` and will note this in the `README.md`.